website/server/models/group.js

```diff
@@ -771,7 +771,7 @@ schema.methods.startQuest = async function startQuest (user) {
 
   // update the users who are not participating
   // Do not block updates
-  User.update({
+  User.updateMany({
     _id: { $in: nonMembers },
   }, _cleanQuestParty(),
   { multi: true }).exec();
```

Comment:
'nother `multi`. Might want to grep these!

Reply:
done!